frontend/src/store/configurate-episode/actions.ts

```diff
@@ -70,4 +70,13 @@ const loadEpisode = createAsyncThunk<Episode, number, AsyncThunkConfig>(
     return episode;
   });
 
-export { create, edit, loadEpisode };
+const deleteEpisode = createAsyncThunk<void, number, AsyncThunkConfig>(
```

Comment:
```suggestion
const delete = createAsyncThunk<void, number, AsyncThunkConfig>(
```

Reply:
Can`t rename, becouse "delete" is native js function.